thirdeye-spi/src/test/java/ai/startree/thirdeye/spi/api/PlanNodeApiTest.java

```diff
@@ -26,7 +29,7 @@ public class PlanNodeApiTest {
   @Test
   public void testInputNode() throws IOException {
     URL resource = PlanNodeApiTest.class.getClassLoader().getResource("inputNode.json");
-    final PlanNodeApi inputNode = new ObjectMapper().readValue(
+    final PlanNodeApi inputNode = ThirdEyeSerialization.newObjectMapper().readValue(
```

Comment:
Seems like `ThirdEyeSerialization.newObjectMapper()` should be injected. Previously this was `static` and it made sense. but now it seems we have a defined way of conducting this everywhere.

Reply:
Will do in a next PR.